content/applications/sales/crm/acquire_leads/opportunities_form.rst

```diff
@@ -0,0 +1,109 @@
+
+In addition to the settings for the form, the settings for each field can be customized as well.
+With the web editor menu still open, click into a field to open the field configuration settings.
+The following options are available to customize a field:
+
+- :guilabel:`Type`: choose a custom field option or an existing field type.
+- :guilabel:`Input Type`: determine the type of information customers should input. Available
+  options are :guilabel:`Text`, :guilabel:`Email`, :guilabel:`Telephone`, or :guilabel:`Url`. The
+  selection made in this field limits the format that customers can use when entering information.
+- :guilabel:`Label`: enter the name for the field.
+- :guilabel:`Position`: choose the way the label is aligned with the rest of the form. The label can
+  be hidden, above the field, to the left of the field, or right adjusted and closer to the field.
+- :guilabel:`Description`: slide the toggle to add a description for the field, which can provide
+  additional instructions to customers. Click under the field on the form to add the description.
+- :guilabel:`Placeholder`: enter an example to guide users how to input information where
+  formatting is important, such as a phone number or email address.
+- :guilabel:`Default Value`: enter a value to include in the form by default if the customer does
+  not provide information in the field. *It is not recommended to include a default value for
+  required fields*.
+- :guilabel:`Required`: slide the toggle to mark this field as required if it **must** be filled in
+  for every submission.
+- :guilabel:`Visibility`: select when this field should be visible.
+- :guilabel:`Animation`: select if this field should have any animation.
+
+.. image:: opportunities_form/field-customization.png
+   :align: center
+   :alt: The field configuration settings on an Odoo website.
+
+View opportunities
+==================
+
+After a customer submits a contact form and an opportunity is created, it is assigned based on the
+:ref:`form settings <crm/customize-contact-form>`. To view opportunities, navigate to
+:menuselection:`'CRM app' --> Sales --> My Pipeline`.
+
+.. note::
+   If leads are activated on the database, contact form submissions are generated as leads, not
+   opportunities. Navigate to :menuselection:`CRM app --> Leads` to view the newly created leads.
+
+On the :guilabel:`My Pipeline` dashboard, click on the Kanban card for an opportunity to open the
+details page. The information submitted by the customer is visible on the opportunity record.
+
+.. note::
+   As the contact form fields are customizable, the fields on the opportunity record where the form
+   information is stored varies.
+
+   If the preconfigured contact form is used, the :guilabel:`Subject` field is added to the
+   :guilabel:`Title` field, and the content in the :guilabel:`Notes` field, which is labeled as
+   :guilabel:`Your Question`, is added to the :guilabel:`Internal Notes` tab.
```

Comment:
If Subject is on the contact form, and is not visible on this screen, I believe the format is to use asterisks. I just had this issue ocme up with Sam when I references a field on a different page form where the user was currently on. It could also be reworded like this:

 If the preconfigured contact form is used, what was entered in the 
 *Subject* field populates the :guilabel:`Title` field,....